include/Metabox/Posts/Category.php

```diff
@@ -92,7 +92,16 @@ public function move( $options ) {
 		foreach ( $posts as $post ) {
 			$current_cats = array_diff( wp_get_post_categories( $post->ID ), array( $options['old_cat'] ) );
 
-			if ( - 1 !== $options['new_cat'] ) {
+			if ( -1 !== $options['new_cat'] ) {
```

Comment:
Instead of repeating the `overwrite` check twice, first check for overwrite condition and inside that check if new cat is `-1` or not.

Reply:
If i check overwrite first then i need to repeat the check cat is -1 or not at overwrite IF and ELSE parts.

Reply:
What I wanted you to do is this.

```
if ( $options['overwrite'] ) {
	// Override is set, so remove all common categories.
	$current_cats = array();
}

if ( -1 !== $options['new_cat'] ) {
	$current_cats[] = $options['new_cat'];
}
```

Compare this with the existing code to see how many if/else conditions we have and also which one is easier to understand.